dist/scripts/binary-dedupe.sh

```diff
@@ -34,10 +34,10 @@ case "$OSTYPE" in
 esac
 
 STEP=0
-export SPARK3XX_COMMON_TXT="$PWD/spark3xx-common.txt"
-export SPARK3XX_COMMON_COPY_LIST="$PWD/spark-common-copy-list.txt"
+export SPARK_COMMON_TXT="$PWD/spark-shared.txt"
+export SPARK_COMMON_COPY_LIST="$PWD/spark-common-copy-list.txt"
```

Comment:
Sorry for nitpicking, but could we pick a lane here?  Is it `shared` or `common`?

Reply:
Done
PTAL